Calendar/Get-RBASummary.ps1

```diff
@@ -310,6 +310,7 @@ function RBADelegateSettings {
     Write-Host "`t ForwardRequestsToDelegates:      "$RbaSettings.ForwardRequestsToDelegates
     Write-Host
 
+    # Warnings:
```

Comment:
You call out a warning but then don't do a `Write-Host` with `-ForegroundColor Yellow`

Reply:
That is just the block header for the next 30 lines of code.  They are all checking for issues.

Updated comments to make it more clear.